tspi/context.go

```diff
@@ -132,3 +132,11 @@ func (context *Context) CreateHash(hash crypto.Hash) (*Hash, error) {
 		hashAlg: hash,
 	}, nil
 }
+
+// GetCapability reads the requested capability and subcapability from the TPM.
+func (context *Context) GetCapability(capa int, subcaplen uint, subcap uint8) ([]byte, error) {
```

Comment:
Can you open this as a different PR?

Also, I know with TPM 2.0 the TPM isn't actually guaranteed to return all the capabilities on the first call. Do you know what the behavior here is? Can we document that?

Can we add constants for the caps and subcaps?

Reply:
The constants should already be in [tspiconst.go](https://github.com/google/go-tspi/blob/master/tspiconst/tspiconst.go).

It looks like all of the TPM 1.2 capabilities are returned as two UINT32 bit maps which are written into a BYTE**. (I can't find any mention as to whether any of the capabilities exceed this size so all the data should be returned per call?)
As far as I can tell the intended use and behaviour is that you call this method for each capability you want--it cannot return multiple capabilities--(if that is what you are asking).

I will move this to a different PR.